docs/user/config-file/v2.rst

```diff
@@ -418,6 +418,42 @@ We don't currently support PPA's or other custom repositories.
    When possible avoid installing Python packages using apt (``python3-numpy`` for example),
    :ref:`use pip or Conda instead <guides/reproducible-builds:pinning dependencies>`.
 
+
+build.jobs
+``````````
+
+Bash commands to be run before or after a Read the Docs pre-defined job.
```

Comment:
```suggestion
Shell commands to be run before or after a Read the Docs pre-defined job.
```

Reply:
Or I suppose even just "Commands", "shell" is probably redundant.

Reply:
Cool, I thought we were calling `/bin/bash -c` but I just saw we are doing `/bin/sh -c`.

https://github.com/readthedocs/readthedocs.org/blob/052f891dbab6b21a5bbdf4570fb149a0adef1581/readthedocs/doc_builder/environments.py#L364